test/integration/create_test.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
 
-const Code = require('code');
-const Lab = require('lab');
+const Code = require('@hapi/code');
+const Lab = require('@hapi/lab');
 const lab = (exports.lab = Lab.script());
 const fs = require('fs');
 const path = require('path');
 const cp = require('child_process');
+const util = require('util');
 const dbmUtil = require('db-migrate-shared').util;
 
-const rmdir = require('rimraf');
+const rmdir = util.promisify(require('rimraf'));
```

Comment:
since we use project wide bluebird, please use bluebird promisify function.

Reply:
I'd advocate for ease of use and less dependencies over, IMO, marginal performance gains - this is specific to the purpose this package serves.  

I'd also note that performance is a function of the node version.  The delta in performance certainly increases as we fallback towards older and older versions of node.

Thoughts on when we can move to use native promisify and promises?